docs/shared/recipes/troubleshoot-cache-misses.md

```diff
@@ -2,7 +2,13 @@
 
 Problem: A task is being executed when you expect it to be replayed from the cache.
 
-1. Check if your task is either marked with `"cache": true` in `nx.json#targetDefaults` or listed in `cacheableOperations` defined in `nx.json#tasksRunnerOptions`
+1. Check if your task is marked as cacheable:
+
+   - Check the task has a "Cacheable" label in the Project Details View. You can do so by running `nx show project <project-name> --web` or by checking the [Project Details View in Nx Console](/recipes/nx-console/console-project-details).
```

Comment:
There's kind of a duplication here where you first mention the label on the Project Details View  and then you mention that again in the last part of the sentence.